citi-onboarding/DADe-onepage
# feat(feature/create-members-section): create OurMembers view

## What I did

- Create OurMembers view and it's necessary components
- Create necessary exports and imports on the index.jsx files

| Mockup | Real |
|--------|------|
![mock](https://user-images.githubusercontent.com/61971582/122260945-3bebbf80-ceaa-11eb-97c8-e22471780693.png) |
![Captura de Tela_select-area_20210616134809](https://user-images.githubusercontent.com/61971582/122260981-4443fa80-ceaa-11eb-9f82-e358daf08c79.png)



## How to test

1. Open the terminal and go to the folder of this project

2. In terminal:
   1. `git checkout develop`
   2. `git pull`
   3. `git checkout feature/create-members-section`
   4. `git pull origin feature/create-members-section`
   5. `npm install`
   6. `cd client/`
   7. `npm start`

3. In your browser go to http://localhost:3000 (should open automatically by `npm start` command)

4. Check if the component looks just like the mockup in different screen sizes (minimum width: 1366px) and if when you increase it past 1920px it remains the same size (Doesnt grow along with the screen).

5. In VScode open the file in the path `client/src/Views/OurMembers/OurMembers.jsx`

6. In the line 7 of the mentioned file, change the imported image to any other image you chose, save the file, and see if the newly added image is displaying in all the cards (all of them are using the same variable as the image, to change a specific one, change any `image: <variable>` between the lines 18 and 29).

7. Between lines 18 and 29, change any of the properties (image, name, cargo and data_inicial) in any of the lines mentioned, save the file, and check if the changes appear on the localhost site.

8. Read the changed files section on GitHub 

9. Don't commit or push your changes from the tests, as the original informations were the correct ones for the current version of the project.

10. Test the things that you think are worth testing, even the ones that are not in this description 

&nbsp;
***

**Resolves:** members issue
